aws/policy/security-services.yaml

```diff
@@ -31,6 +31,7 @@ Statement:
           - 'arn:aws:iam::aws:policy/service-role/AmazonDMSVPCManagementRole'
           - 'arn:aws:iam::aws:policy/service-role/AmazonRDSEnhancedMonitoringRole'
           - 'arn:aws:iam::aws:policy/service-role/AWSServiceRoleForVPCTransitGateway'
+          - 'arn:aws:iam::aws:policy/service-role/AmazonEC2ContainerServiceRole'
```

Comment:
This one is duplicated, and probably should be `arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy`